codec-http/src/test/java/io/netty/handler/codec/http/multipart/AbstractMemoryHttpDataTest.java

```diff
@@ -74,22 +74,34 @@ private TestHttpData(String name, Charset charset, long size) {
 
         @Override
         public InterfaceHttpData.HttpDataType getHttpDataType() {
-            throw new UnsupportedOperationException("Should never be called.");
+            reject();
+            return null;
         }
 
         @Override
         public HttpData copy() {
-            throw new UnsupportedOperationException("Should never be called.");
+            return reject();
         }
 
         @Override
         public HttpData duplicate() {
-            throw new UnsupportedOperationException("Should never be called.");
+            return reject();
+        }
+
+        @Override
+        public HttpData retainedDuplicate() {
+            return reject();
+        }
+
+        @Override
+        public HttpData replace(ByteBuf content) {
+            return null;
         }
 
         @Override
         public int compareTo(InterfaceHttpData o) {
-            throw new UnsupportedOperationException("Should never be called.");
+            reject();
+            return 0;
```

Comment:
Let me make `reject()` return an exception so that it becomes `throw reject()` then.